Formula/vault.rb

```diff
@@ -27,14 +27,14 @@ class Vault < Formula
 
   depends_on "go" => :build
   depends_on "gox" => :build
-  depends_on "node" => :build
-  depends_on "python@3.10" => :build
+  depends_on "node@18" => :build
+  depends_on "python@3.11" => :build
```

Comment:
Seems like version of gyp used is incompatible with Python 3.11

Reply:
Current `vault` release 1.12.2 uses `node-gyp == 7.1.2`. The `main` branch uses `node-gyp == 8.4.1`.

Issue should be fixed with `node-gyp >= 8.0.0`.